src/ui/widgets/EmbeddedDisplay/opiParser.ts

```diff
@@ -176,6 +194,22 @@ export function opiParseActions(
               (action.description && action.description._text) || undefined
           }
         });
+      } else if (type === OPEN_DISPLAY) {
+        processedActions.actions.push({
+          type: OPEN_DISPLAY,
+          dynamicInfo: {
+            name: action.path._text,
+            location: descriptionToLocation(action),
+            description:
+              (action.description && action.description._text) || undefined,
+            file: {
+              path: action.path._text,
+              // TODO: Should probably be accessing properties of the element here
+              macros: {},
+              defaultProtocol: "pva"
+            }
+          }
+        });
```

Comment:
The reason the new pages aren't connecting to coniql, could be because I am only assigning a blank object to the macros, but am not sure how to access the correct macros from this point in the code.